sdk/core/Azure.Core/src/Response.cs

```diff
@@ -82,6 +82,14 @@ public virtual BinaryData Content
         /// </summary>
         public abstract void Dispose();
 
+        /// <summary>
+        /// Indicates whether the message's <see cref="ResponseClassifier"/> considers this
+        /// response an error.
+        /// </summary>
+        public bool IsError { get; internal set; }
+
+        internal HttpMessageSanitizer? Sanitizer { get; set; }
```

Comment:
We need to switch all Core consumers to use this property, do you mind filing an issue?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/25555